app.json

```diff
@@ -13,6 +13,7 @@
   "buildpacks": [
   ],
   "env": {
+    "IMMS_DATA_SERVER": "https://fake-immunisation-data-server.herokuapp.com/"
```

Comment:
```suggestion
    "IMMS_DATA_SERVER": {
       "required": true,
       "value": "https://fake-immunisation-data-server.herokuapp.com/"
       }
```

Reply:
BTW: i think this will only populate into ENV on new apps - we'll need to set it manually on production.